src/components/Integrations/Shopify/Shopify.js

```diff
@@ -82,7 +90,7 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
                 <div className="block dp-integration__status">
                   <div className="status__info">
                     <div>
-                      <img className="shopify-logo" src={logo} alt="Shopify logo" />
+                      <StyledShopifyLogo svglogo={logo} />
```

Comment:
The idea is use the word "Styled" first when use styled components?

Reply:
Yes this way is suggested in Styled Components page. To keep a standard, because we have two components here: `ShopifyLogo` and `StyledShopifyLogo`

Reply:
It's the way I found, if you can think of a better one, please tell me.